packages/theme/template/package.json

```diff
@@ -41,7 +41,7 @@
     "ilib": "^14.14.0 || ^14.14.0-webos1"
   },
   "devDependencies": {
-    "eslint-config-enact-proxy": "^1.0.4",
-    "ilib": "^14.14.0"
+    "eslint-config-enact-proxy": "^1.0.5",
```

Comment:
Note to reviewer: Do we need to update  "eslint-config-enact-proxy" version when the new version is released?